### PR TITLE
Add tests to client/app/services/vm-details/vm-details.component.js

### PR DESCRIPTION
**Pull request by chalettu**
PV https://www.pivotaltracker.com/story/show/145296867

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label test
@miq-bot add_label fine/no
@miq-bot add_label technical debt

**Comment by chalettu** [comment]
No UX changes.  Just adding tests

File: client/app/services/vm-details/vm-details.component.js [hunk 1]
```diff
@@ -35,9 +41,11 @@ function ComponentController($stateParams, VmsService, ServicesState, sprintf, l
           actionsInclude: true,
         },
       },
+      listActions: [],
     });
-    EventNotifications.info(__("The contents of this page is a function of the current users's group."));
     resolveData();
+    EventNotifications.info(__("The contents of this page is a function of the current users's group."));
```
Comment (AllenBW) [inline client/app/services/vm-details/vm-details.component.js]: Why move this notification?

Comment (chalettu) [inline client/app/services/vm-details/vm-details.component.js]: I had reformatted some other things in the file.  I can move it back.  I don't think its a big deal either way.

Comment (AllenBW) [inline client/app/services/vm-details/vm-details.component.js]: Agreed, not a big deal, was just wondering if there was a specific reason... OOOOO TIME FOR REBASE ANYWAYS :p

@chalettu

**Pull request merged**